### PR TITLE
DESCRIPTION update

### PR DESCRIPTION
**Pull request by christophsax**

### CODE REVIEW

**Comment by christophsax** [comment]
Had to login to appveyor and delete the run. Seemed to work.

File: DESCRIPTION [hunk 1]
```diff
@@ -1,8 +1,7 @@
 Package: x13binary
 Type: Package
 Title: Provide the 'x13ashtml' Seasonal Adjustment Binary
-Version: 1.1.57-4
-Date: 2023-09-07
+Version: 1.1.60
```
Comment (eddelbuettel) [inline DESCRIPTION:4]: Date is indeed optional. I find it helpful.  Your call.

**Review by eddelbuettel — APPROVED** [review]
Looks good.

File: DESCRIPTION (hunk 1 above)
Comment (christophsax) [reply DESCRIPTION:4]: It saves me from forgetting to update

**Pull request merged**